spec/factories/role_factory.rb

```diff
@@ -2,7 +2,7 @@
 
 FactoryBot.define do
   factory :role do
-    name { Faker::Construction.role }
+    name { Faker::Lorem.unique.characters(number: 10) }
```

Comment:
It's either that or we clear the cache in every single spec.

![image](https://user-images.githubusercontent.com/43917914/195439567-0843e5f1-4e29-448e-a212-fabe4e549eaf.png)

Reply:
Eh after all it might be worth it to clear the cache in every file. I prefer the Construction role name as it helps a lot when debugging.

Reply:
Change Construction.role to Job.title which has the same effect but has a bigger range.